src/main/scala/djinni/Main.scala

```diff
@@ -87,9 +87,12 @@ object Main {
     var yamlOutFile: Option[String] = None
     var yamlPrefix: String = ""
     var pyOutFolder: Option[File] = None
-    var pyPackageName: String = ""
     var pyIdentStyle = IdentStyle.pythonDefault
     var cWrapperOutFolder: Option[File] = None
+    var cWrapperHeaderOutFolderOptional: Option[File] = None
+    var cWrapperIncludePrefix: String = ""
+    var cWrapperIncludeCppPrefix: String = ""
+    var cWrapperBaseLibIncludePrefix: String = "djinni/cwrapper/"
```

Comment:
The base-lib now has a default prefix of `djinni/cwrapper`. This should work with the existing support-lib.